status-page.py

```diff
@@ -1,72 +1,133 @@
 import json
 import sys
+from urllib.parse import urlparse
+
 import requests
 
 
 class StatusPage:
     def __init__(self):
         config_data_file = open(sys.argv[1])
-        self.test_results = open(sys.argv[2])
+        log_file = open(sys.argv[2])
```

Comment:
As I mentioned in the previous PR, we can probably rename this file to `api-status.json` to avoid confusion. Also, I recommend using this method to parse aruguments: https://github.com/osu-mist/express-api-skeleton/blob/master/src/tests/integration/utils.py#L15-L35